recipes/forest_lite/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "forest_lite" %}
+{% set version = "0.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MetOffice/forest-lite/archive/v{{ version }}.tar.gz
+  sha256: 24103013a84aad2303986186ce6d328dcabf3e659a81ed88919de9328f12ecfe
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Can you add the entrypoint here, that will probably fix the Windows build.

Reply:
@ocefpaf what is the syntax for adding an entrypoint?

Reply:
Sorry, should've pasted it here. Something like this:

https://github.com/conda-forge/ipython-feedstock/blob/master/recipe/meta.yaml#L15-L16

PS: if you use [tools like grayskull](https://github.com/conda-incubator/grayskull) to generate your recipe this should be added automatically. However, I'm not sure grayskull support GitHub sources yet. I believe it is only PyPI at the moment.

Reply:
@ocefpaf is it better to build from PyPI than GitHub? Like, is it best practice?